tomviz/MainWindow.cxx

```diff
@@ -501,6 +501,15 @@ void MainWindow::showEvent(QShowEvent* e)
 
 void MainWindow::closeEvent(QCloseEvent* e)
 {
+  if (ModuleManager::instance().hasRunningOperators()) {
+    QMessageBox::StandardButton response = QMessageBox::question(
+      this, "Close tomviz?", "You have transforms that are not completed.  Are "
```

Comment:
Just use on space after a full stop, if they click yes is there any hope of exiting cleanly? If not, then maybe we should consider warning about that.